monkey/infection_monkey/pyinstaller_hooks/hook-infection_monkey.ransomware.py

```diff
@@ -0,0 +1,4 @@
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules
+
+hiddenimports = collect_submodules("infection_monkey.ransomware")
+datas = collect_data_files("infection_monkey.ransomware", include_py_files=True)
```

Comment:
Why is this needed?

Reply:
Because the ransomware_readme.txt file is not included in the binary by default. We need to provide PyInstaller with these hooks so it knows to collect data files, not just python files.